src/components/FileTree/index.spec.tsx

```diff
@@ -389,5 +389,19 @@ describe(__filename, () => {
 
       expect(node.props.onSelect).toEqual(onSelect);
     });
+
+    it('passes the version ID to FileTreeNode', () => {
+      const version = getVersion({ ...fakeVersion, id: 777 });
+
+      const root = render({ version });
+
+      const node = (root.instance() as FileTree).renderNode(
+        // It does not really matter which props are given here.
+        // eslint-disable-next-line @typescript-eslint/no-object-literal-type-assertion
+        {} as TreefoldRenderProps<TreeNode>,
```

Comment:
Why not just get the props using the helper that already exists?

```diff
diff --git a/src/components/FileTree/index.spec.tsx b/src/components/FileTree/index.spec.tsx
index faae880..6ccce01 100644
--- a/src/components/FileTree/index.spec.tsx
+++ b/src/components/FileTree/index.spec.tsx
@@ -13,6 +13,7 @@ import {
 } from '../../reducers/versions';
 import { fakeVersion, fakeVersionEntry } from '../../test-helpers';
 import { getLocalizedString } from '../../utils';
+import { getTreefoldRenderProps } from '../FileTreeNode/index.spec';
 
 import FileTree, { buildFileTree, TreeNode, DirectoryNode } from '.';
 
@@ -396,9 +397,7 @@ describe(__filename, () => {
       const root = render({ version });
 
       const node = (root.instance() as FileTree).renderNode(
-        // It does not really matter which props are given here.
-        // eslint-disable-next-line @typescript-eslint/no-object-literal-type-assertion
-        {} as TreefoldRenderProps<TreeNode>,
+        getTreefoldRenderProps(),
       );
 
       expect(node.props.versionId).toEqual(version.id);
diff --git a/src/components/FileTreeNode/index.spec.tsx b/src/components/FileTreeNode/index.spec.tsx
index ef47651..996697a 100644
--- a/src/components/FileTreeNode/index.spec.tsx
+++ b/src/components/FileTreeNode/index.spec.tsx
@@ -9,39 +9,39 @@ import styles from './styles.module.scss';
 
 import FileTreeNode, { FileTreeNodeBase, PublicProps } from '.';
 
-describe(__filename, () => {
-  const fakeGetToggleProps = () => ({
-    onClick: jest.fn(),
-    onKeyDown: jest.fn(),
-    role: 'button',
-    tabIndex: 0,
-  });
+const fakeGetToggleProps = () => ({
+  onClick: jest.fn(),
+  onKeyDown: jest.fn(),
+  role: 'button',
+  tabIndex: 0,
+});
 
-  const getTreefoldRenderProps = ({
-    getToggleProps = fakeGetToggleProps,
-    hasChildNodes = false,
-    id = 'root',
-    isExpanded = false,
-    isFolder = true,
-    name = 'root',
-    renderChildNodes = jest.fn(),
-    onSelect = jest.fn(),
-  } = {}) => {
-    return {
-      node: {
-        id,
-        name,
-      },
-      getToggleProps,
-      hasChildNodes,
-      isExpanded,
-      isFolder,
-      level: 0,
-      renderChildNodes,
-      onSelect,
-    };
+export const getTreefoldRenderProps = ({
+  getToggleProps = fakeGetToggleProps,
+  hasChildNodes = false,
+  id = 'root',
+  isExpanded = false,
+  isFolder = true,
+  name = 'root',
+  renderChildNodes = jest.fn(),
+  onSelect = jest.fn(),
+} = {}) => {
+  return {
+    node: {
+      id,
+      name,
+    },
+    getToggleProps,
+    hasChildNodes,
+    isExpanded,
+    isFolder,
+    level: 0,
+    renderChildNodes,
+    onSelect,
   };
+};
 
+describe(__filename, () => {
   type RenderParams = Partial<PublicProps> & { store?: Store };
 
   const render = ({
```

Reply:
ah, because I am stupid I suppose 🤦‍♂️